qiime2/plugin/__init__.py

```diff
@@ -14,12 +14,11 @@
                               MetadataColumn, Categorical, Numeric, Properties,
                               Range, Start, End, Choices, Bool, Set, List,
                               Visualization, TypeMap, TypeMatch)
-from qiime2.sdk.usage import UsageAction, UsageInputs, UsageOutputNames
 
 
 __all__ = ['TextFileFormat', 'BinaryFileFormat', 'DirectoryFormat', 'Plugin',
            'SemanticType', 'Set', 'List', 'Bool', 'Int', 'Str', 'Float',
            'Metadata', 'MetadataColumn', 'Categorical', 'Numeric',
            'Properties', 'Range', 'Start', 'End', 'Choices', 'Visualization',
            'TypeMap', 'TypeMatch', 'ValidationError', 'Citations',
-           'CitationRecord', 'UsageAction', 'UsageInputs', 'UsageOutputNames']
```

Comment:
The `plugin` module is where plugin devs expect to import "plugin" related things. Similarly, the `sdk` module is for sdk-devs. We can't go removing the exports from the `plugin` module, because that breaks the plugin-dev contract. One option is to move the `usage` module into a new location and export/expose in `plugin` and `sdk`. A candidate here might be `core` (ccing @ebolyen). You might also be able to flip-flop, and move `usage` into plugin and expose the relevant giblets via `sdk`.

Reply:
Moving usage into `plugin` and exposing the giblets in `sdk` seems to work just fine!  🦃   Do you have a preference WRT keeping this changeset distinct from #546?

Reply:
lol, glad to hear the giblet are doing well! no preference - just let me know what you want to do!

Reply:
Let's close this PR and I'll include this changeset in #546.